### ISSUE TITLE
cannot read tcx

### ISSUE THREAD

**Issue by skotturi** [jmackie4/cycleRtools#3]
when running `> tcx_parse = read_tcx("my_file.tcx")`
I'm getting: 

```
xmlXPathEval: evaluation failed
 Hide Traceback
```

The error is being generated by: 

```
> if (any(grepl("Extensions", cols))) {
+     cols <- cols[!grepl("Extensions", cols)]
+     tmp <- suppressWarnings(xml_name(xml_children(xml_find_one(x, 
+                                                                "//ns3:TPX", ns))))
+     cols <- c(cols, paste0("//ns3:", tmp))
+ }
```

couple variables before this code block was run: 
```
> x
{xml_document}
<TrainingCenterDatabase>
[1] <Activities>\n    <Activity Sport="Biking">\n      <Id>2015-04-24T07:07:43.000Z</Id>\n      <Lap StartTime="2015-04-24T07:0 ...
[2] <Author xsi:type="Application_t">\n    <Name>tapiriik</Name>\n    <Build>\n      <Version>\n        <VersionMajor>0</Versio ...
> cols
[1] "//d1:Time"             "//d1:AltitudeMeters"   "//d1:DistanceMeters"   "//d1:HeartRateBpm"     "//d1:Cadence"         
[6] "//d1:Extensions"       "//d1:LatitudeDegrees"  "//d1:LongitudeDegrees"
> ns
tpx <-> http://www.garmin.com/xmlschemas/ActivityExtension/v2
ns4 <-> http://www.garmin.com/xmlschemas/ProfileExtension/v1
ns2 <-> http://www.garmin.com/xmlschemas/UserProfile/v2
xsi <-> http://www.w3.org/2001/XMLSchema-instance
ns5 <-> http://www.garmin.com/xmlschemas/ActivityGoals/v1
d1  <-> http://www.garmin.com/xmlschemas/TrainingCenterDatabase/v2
```

Guessing it has something to do with this file containing ns2, ns4 and ns5 namespaces but not ns3 as the function is looking for?

**Comment by jmackie4** [comment]
Hi,

Would you mind sending me the file that raised the error?

Thanks

Jordan

**Comment by skotturi** [comment]
yea, for sure. I don't see an email address in your profile, where's a good place for me to send it?

**Comment by jmackie4** [comment]
Hi, send it over to jmackie at protonmail dot com. Cheers.

**Comment by jmackie4** [comment]
Fixed as of 710aa0832c2201228200ad2e04d4f0985cff43e3. Thanks
